### PR TITLE
[CPU] Get rid of shouldTryBrgconv flag

### PR DESCRIPTION
**Pull request by EgorDuplensky**
### Details:
 - No code logic changed overall
 
 At the moment shouldTryBrgConv is essentially just a check for avx512 isa.

- [ ] Run xeon benchmark just in case

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@usstq  @luo-cheng2021 guys, any concerns from your side?

File: src/plugins/intel_cpu/src/nodes/conv.cpp [hunk 1]
```diff
@@ -560,10 +537,10 @@ void Convolution::getSupportedDescriptors() {
     auto outputShape = getOutputShapeAtPort(0);
 
 #if defined(OPENVINO_ARCH_X86_64)
-    bool acceptedFormat = inputDataType == memory::data_type::bf16;
+    // nspc shows better performance only whit brgconv implementation
```
Comment (luo-cheng2021) [inline src/plugins/intel_cpu/src/nodes/conv.cpp:540]: typo: whit->hitting.

**Comment by luo-cheng2021** [comment]
LGTM.

**Review by luo-cheng2021 — APPROVED** [review]

File: src/plugins/intel_cpu/src/nodes/conv.cpp [hunk 2]
```diff
@@ -997,11 +975,11 @@ void Convolution::SetPostOpsAndZeroPoints(std::vector<dnnl::primitive_attr> &att
...
-        DEBUG_LOG(getName(), ": shouldTryBrgconv = false. Skip extra attribute");
+    if (!isBrgConvAvailable) {
+        DEBUG_LOG(getName(), ": brgconv is not available. Skip extra attribute");
         return;
     }
-    // Try 2 attributes. Consider the shouldTRyBrgconv could be set via RTinfo to enforce brgconv.
+    // Try 2 attributes.
     attrs.resize(2);
     if (inputZeroPointType == zpType::PerTensor && dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx)) {
         //WR to ONEDNN limitation. attr[1] - legacy post ops + stock zero point.
```
Comment (luweizhou2016) [inline src/plugins/intel_cpu/src/nodes/conv.cpp:985]: BTW. Seems ONEDNN 3.2 would support zp on brgconv amx.  Hope it also can be removed in next release:-)

**Review by luweizhou2016 — APPROVED** [review]

File: src/plugins/intel_cpu/src/nodes/conv.h [hunk 3]
```diff
@@ -156,7 +156,7 @@ class Convolution : public Node {
     const size_t Y_AXIS = 1;
 
     bool isWino = false;
-    bool shouldTryBrgconv = false;
+    bool isBrgConvAvailable = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core);
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/conv.h:159]: static const

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**